build.sbt

```diff
@@ -184,20 +156,15 @@ lazy val communityBuildProjects: Seq[ProjectReference] = Seq(
   airspecJVM
 )
 
-// JVM projects supporting Scala 2.11 - Scala 2.13
+// Other JVM projects supporting Scala 2.12 - Scala 2.13
 lazy val jvmProjects: Seq[ProjectReference] = communityBuildProjects ++ Seq[ProjectReference](
   jdbc,
   fluentd,
   airspecLight,
   finagle,
   okhttp,
-  httpRecorder
-)
-
-// JVM projects excluded from Scala 2.13 build
-lazy val jvmProjects2_12: Seq[ProjectReference] = Seq(
+  httpRecorder,
   sql,
-  benchmark,
```

Comment:
Is it okay to remove benchmark project from jvmProjects?

Reply:
Good catch. I forgot about adding it. Thanks